options.go

```diff
@@ -450,8 +453,15 @@ func (o *ClientOptions) SetCustomOpenConnectionFn(customOpenConnectionFn OpenCon
 }
 
 // SetAutoAckDisabled enables or disables the Automated Acking of Messages received by the handler.
+//
 //	By default it is set to false. Setting it to true will disable the auto-ack globally.
 func (o *ClientOptions) SetAutoAckDisabled(autoAckDisabled bool) *ClientOptions {
 	o.AutoAckDisabled = autoAckDisabled
 	return o
 }
+
+// SetLogLevel sets the log level for the client. This will be used to control the verbosity of the logs
+func (o *ClientOptions) SetLogLevel(logVerbosity LogLevel) *ClientOptions {
```

Comment:
As commented elsewhere I think `SetLogger(l *slog.Logger) *ClientOptions` might be more flexible? 

A few reasons:
* One of the main belefits of using `slog` is that it's part of the standard library and there are heaps of adaptors (for example I use `zerolog` in quite a few apps and could use `https://github.com/samber/slog-zerolog` to get an `slog.Logger` to pass into such a function).
* Users can use whatever output type they want